### PR TITLE
Fuzzer st with different t ms

### PR DESCRIPTION
**Pull request by comeonbuddy**
### Please check if the PR fulfills these requirements
- [ x] The commit message follows our Submission guidelines
- [ x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? 
(Bug fix, feature, docs update, ...)

### What is the current behavior? 
(You can also link to an open issue here)

### What is the new behavior?
(Define and describe any new functionality. Clarify if this is a feature change)

### Does this PR introduce a breaking change?
(What changes might users need to make in their application due to this PR?)

### CODE REVIEW

File: contracts/tokens/SecurityToken.sol [hunk 1]
```diff
@@ -531,7 +531,7 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
      * @return bool success
      */
     function transferWithData(address _to, uint256 _value, bytes _data) public returns (bool success) {
-        require(_updateTransfer(msg.sender, _to, _value, _data), "Transfer invalid");
+        // require(_updateTransfer(msg.sender, _to, _value, _data), "Transfer invalid");
```
Comment (satyamakgec) [inline contracts/tokens/SecurityToken.sol]: what is the reason to comment this statement?

**Review by satyamakgec — CHANGES_REQUESTED** [review]

File: contracts/tokens/SecurityToken.sol (hunk 1 above)
Comment (comeonbuddy) [reply contracts/tokens/SecurityToken.sol]: sorry I forgot to recover this, it was comment off for debugging :/ updated.

**Comment by satyamakgec** [comment]
@comeonbuddy Travis and CircleCI both are failing could you please take a look.

**Comment by satyamakgec** [comment]
I don't know why @comeonbuddy added the same test file twice. I am closing this PR. It has no use.

**Pull request closed without merging**